wire-ios/Wire-iOS Tests/Bundle/URL+WireTests.swift

```diff
@@ -39,7 +39,6 @@ final class URL_WireTests: XCTestCase {
     }
 
     func testThatAccountURLsAreLoadedCorrectly() {
-        let accountsURL = URL(string: "https://account.wire.com")!
-        XCTAssertEqual(be.accountsURL, accountsURL)
+        XCTAssertEqual(WireURLs.shared.passwordReset, be.accountsURL.appendingPathComponent("forgot"))
```

Comment:
1. keep old test and add a new test for this one as test_passwordReset_URLIsCorrect
2. test_passwordReset_URLIsCorrect should be added to the securityTests plan